### ISSUE TITLE
justMycode = false does not work

### ISSUE THREAD

**Issue by flyforward2016** [microsoft/debugpy#795]
### VS Code version

1.62.3 

### Extension version

2021.11.1422169775

### OS type

Windows

### OS version

Windows_NT x64 10.0.19043

### Python distribution

Anaconda

### Python version

3.6.4

### Language server

Pylance

### Expected behaviour

This is an issue that initially posted to vscode but was asked to post in this forum (vscode - python). 

I have been trying to change the justMycode from default "true" to "false", to debug into pytorch package but with no luck.

I have added the following code in the configurations( the second parenthesis is the newly added, the 1st one was by default). It did not work on 1.59 nor the most updated 1.62. What have gotten worse was the "python" after "type" in the configuration was not recognized anymore after upgrading from 1.59 to 1.62. The error was about "Configured debug type 'python' is installed but not supported in this environment."

PS: the code I used in the configuration was followed in this link https://code.visualstudio.com/docs/python/testing#_debug-tests

Please let me know if you could point me to the right direction and resolve the issue. Thank you in advance.

```
"version": "0.2.0",
"configurations": [
{
"name": "Python: Current File",
"type": "python",
"request": "launch",
"program": "${file}",
"console": "integratedTerminal",
},
{
"name": "Python: Current File",
"type": "python",
"request": "launch",
"program": "${file}",
"purpose": ["debug-test"],
"console": "integratedTerminal",
"justMyCode": false
}
]
}
```

### Actual behaviour

When debugging the code, I should be able to step into the Pytorch library if I set "JustMyCode" to false. Please refer to expected behavior for more details

### Steps to reproduce

Please see expected behavior for more detail. 

Basically I was just changing the lauch.json file as below, which should turn off the "JustMyCode" function but it did not. 
```

"version": "0.2.0",
"configurations": [
{
"name": "Python: Current File",
"type": "python",
"request": "launch",
"program": "${file}",
"console": "integratedTerminal",
},
{
"name": "Python: Current File",
"type": "python",
"request": "launch",
"program": "${file}",
"purpose": ["debug-test"],
"console": "integratedTerminal",
"justMyCode": false
}
]
}
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by int19h** [comment]
The "not supported in this environment" issue seems to be a dupe of https://github.com/microsoft/vscode-python/issues/17952. Per that issue, it is already fixed in VSCode Insiders - can you please try that?

**Comment by flyforward2016** [comment]
Hi Pavel,

After updating the insider version, that has been resolved.

Thanks,

Bin

**Comment by Vishnu-Naik** [comment]
Recently, even I faced this issue where the VS code was not taking the latest launch.json (one with 'justMycode: false'). So, I had to perform the following steps.

1. Instead of running a debugger from the Top right group menu of the editor, I ran it from below status bar as shown in the below picture

![image](https://user-images.githubusercontent.com/41805841/201196032-6716e5d4-1550-4764-abb1-6204cb704003.png)

2. Once clicked on this option a pop-up appears asking which launch.json you want to run your debugging with, as shown below.

![image](https://user-images.githubusercontent.com/41805841/201196806-23b38893-221a-4e72-989c-56e7c4b77dc4.png)

You can click on launch.json here to edit the configuration file and now the debugger opens with the selected launch.json and 'justtMycode:false' setting will be applied.

**Comment by wangzhaode** [comment]
> Recently, even I faced this issue where the VS code was not taking the latest launch.json (one with 'justMycode: false'). So, I had to perform the following steps.
> 
> 1. Instead of running a debugger from the Top right group menu of the editor, I ran it from below status bar as shown in the below picture
> 
> ![image](https://user-images.githubusercontent.com/41805841/201196032-6716e5d4-1550-4764-abb1-6204cb704003.png)
> 
> 2. Once clicked on this option a pop-up appears asking which launch.json you want to run your debugging with, as shown below.
> 
> ![image](https://user-images.githubusercontent.com/41805841/201196806-23b38893-221a-4e72-989c-56e7c4b77dc4.png)
> 
> You can click on launch.json here to edit the configuration file and now the debugger opens with the selected launch.json and 'justtMycode:false' setting will be applied.

thanks! 1.85.1 has this problem and find nothong helpful solution util see your comments.

**Comment by Keneyr** [comment]
@Vishnu-Naik my vscode doesn't have this "Python: Current File(CodeBase) "UI....
![image](https://github.com/microsoft/debugpy/assets/25053570/8e1f29d0-b13c-4c5c-9cd9-89fcd3743e98)

**Comment by Celica31** [comment]
it works for me,thank you guys so much.

**Comment by rchiodo** [comment]
@Aliwoto, VS code has a bug in 1.90 that is causing your problem. See the following issue:
https://github.com/microsoft/debugpy/issues/1596
